src/js/components/DataTable/doc.js

```diff
@@ -233,6 +241,13 @@ export const doc = DataTable => {
       where the background, border, and pad accept the same values as
       the same named properties on DataTable.`,
     ),
+    select: PropTypes.arrayOf(
+      PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    ).description(
+      `Which rows should be selected. The values in this array should match
+      the primaryKey or columns[].primary keyed value for the row's data
+      object.`,
```

Comment:
Should we add something about the control of the selected rows by using `onSelect`?

Reply:
added